packages/compiler/src/collector/collect.ts

```diff
@@ -50,6 +63,28 @@ const extractAttribute = (jsxAttribute: JsxAttribute) => {
         const expression = initializer.getExpression();
         if (!expression) return;
 
+        // fontSize={... ? ... : ...}
+        const conditionalExpression = match(expression)
+          .when(Node.isConditionalExpression, (conditional) => {
```

Comment:
I actually prefer this logic to be placed in `expression.ts` because, as a reader, it's more intuitive to have all expressions handled in `expression.ts`

Reply:
Sure, the initial reason why I put it here is to limit the scope of the change; but at this point probably better to clean and unify the logic there. Let me try a little bit refactoring although it will affect more such as pseduo props handling, which might be actually desired as long as.... it works without unexpected bug :)

Reply:
@poteboy, now I updated this PR to include some refactorings and pseudo props should be also handled properly for conditional expression (cc @kotarella1110) 👍